src/queries/role.js

```diff
@@ -153,7 +153,7 @@ function queryAllByRole(
         }),
         element,
         name,
-        text => text,
+        matchNormalizer,
```

Comment:
What else does `matchNormalizer` do? If you just want to fix `nbsp` then we should make this explicit.

Reply:
According to request at issue `matchNormalizer` - here sync behavior for `ByLabelText` and `ByRole`.
the idea that we'll have the same behavior/results for the same inputs.